docs/azure_arc_jumpstart/azure_arc_sqlsrv/azure/azure_arm_template_winsrv/_index.md

```diff
@@ -39,13 +39,15 @@ By the end of the guide, you will have an Azure VM installed with Windows Server
 
```

Comment:
The readme needs to follow the scenario write up guidelines. Just a few things I've noticed:
 
- Scenario should start with "The following scenario" not "The following readme"
- use dash instead of asterisk for bullets

Reply:
@dkirby-ms and I worked through this together on a call and have applied the necessary code changes.